src/server/init.ts

```diff
@@ -50,6 +49,7 @@ import {getMalloyDefinitionReference} from './definitions/definitions';
 import {TranslateCache} from './translate_cache';
 import {ConnectionManager} from '../common/connection_manager';
 import {findMalloyLensesAt} from './lenses/lenses';
+import {TOKEN_MODIFIERS, TOKEN_TYPES, stubMalloyHighlights} from './highlights';
```

Comment:
Looking more carefully, all of this can go. Both the
```typescript
        semanticTokensProvider: {
          full: true,
          range: false,
          legend: {
            tokenTypes: TOKEN_TYPES,
            tokenModifiers: TOKEN_MODIFIERS,
          },
```
and
```
  connection.languages.semanticTokens.on(handler => {
    const document = documents.get(handler.textDocument.uri);
    return document && document.languageId === 'malloy'
      ? stubMalloyHighlights(document)
      : new SemanticTokensBuilder().build();
  });
```
below can be cut, since they're not configured to actually provide tokens.